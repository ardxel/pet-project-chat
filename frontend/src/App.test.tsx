import App from './App';
import { create } from 'react-test-renderer';

describe('first testing', () => {
  test('test main component', () => {
    const component = create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
