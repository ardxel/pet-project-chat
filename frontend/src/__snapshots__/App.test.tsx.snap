// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first testing test main component 1`] = `
<div
  className="text-red-800"
>
  Hello world!!!
</div>
`;
